python/GafferUI/PresetsPlugValueWidget.py

```diff
@@ -57,11 +57,7 @@ def __init__( self, plugs, **kw ) :
 		self.__menuButton = GafferUI.MenuButton( "", menu = GafferUI.Menu( Gaffer.WeakMethod( self.__menuDefinition ) ) )
 		cont.addChild( self.__menuButton )
 
-		self.__customValuePlugWidget = GafferUI.PlugValueWidget.create( plugs, typeMetadata = None )
-		if type( self.__customValuePlugWidget ) == GafferUI.ConnectionPlugValueWidget and len( self.getPlugs() ) and next( iter( self.getPlugs() ) ).children():
```

Comment:
There's no real mention here of what this special case for ConnectionPlugValueWidget was, or why it's OK to remove it now - you've probably considered it, but it's not clear from looking at the PR.

Reply:
Ah, I'd hoped that including the ImageWriterUI metadata change in the same commit made that clear. The original special case here was added to support the ImageWriterUI, where a presets widget is applied to a plug with children. It basically says "if the widget we got was the default one, and the plug has children, then use a LayoutPlugValueWidget". I've removed that hardcoding, and used `presetsPlugValueWidget:customWidgetType` in ImageWriterUI to preserve the behaviour in the one place we were using it. But the big benefit is that now _anyone_ can specify _any_ custom widget.